dev-client/src/store/sync/hooks/syncHooks.tsx

```diff
@@ -71,3 +77,21 @@ export const useRetryInterval = (interval: number, action: () => void) => {
 
   return {beginRetry, endRetry};
 };
+
+export const useSyncErrorSiteIds = () => {
+  return Object.keys(useSelector(selectSyncErrorSites));
```

Comment:
This will return a new array for every render; better to just create a new memoized selector with `createSelector` in the soil selectors file.

Reply:
Ah, thanks for catching that!
The [Redux docs](https://redux.js.org/usage/writing-tests#:~:text=Do%20not%20try%20to%20mock,actual%20app%20code%20is%20working.) advise not mocking selector functions in tests (though mocking this hook that just wraps a redux selector may also not be in the spirit of the advice...?)
I'll plan to add the new memoized selector, and use it in `useSyncErrorSiteIds` if that sounds good?